docs/versioned-plugins/inputs/elastic_agent-index.asciidoc

```diff
@@ -0,0 +1,19 @@
+:plugin: elastic_agent
+:type: input
+
+[id="{type}-{plugin}-index"]
+
+== Versioned {plugin} {type} plugin
+[subs="attributes"]
+++++
+<titleabbrev>{plugin}</titleabbrev>
+++++
+
+The `input-elastic_agent` plugin is the next generation of the
+<<input-beats-index,input-beats plugin>>, and is based on the
+https://github.com/logstash-plugins/logstash-input-beats[beats repository].
+
+<<input-beats-index,Beats plugins>> versioned v.6.1.3 and higher have an {type}-{plugin} counterpart.
```

Comment:
```suggestion
<<input-beats-index,Beats plugins>> versioned v6.1.3 and higher have an {type}-{plugin} counterpart.
```

Reply:
@karenzone maybe I'm not understanding but what is the goal with this sentence?

Reply:
The input-elastic_agent didn't exist before v6.1.3.  We're linking to the input-beats versions page, but earlier versions aren't applicable. I'll work on rewording to make this clearer.

Reply:
I removed that sentence. If a user runs `run bin/logstash-plugin list --verbose`, no versions earlier than v6.1.3 will be returned anyway. If they're looking for the installed version, then all should be well.